src/Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp

```diff
@@ -101,16 +102,18 @@ struct ComputeSources {
           Tags::TildeS<Frame::Inertial, NeutrinoSpecies>>&... tilde_s,
       const db::const_item_type<
           Tags::TildeP<Frame::Inertial, NeutrinoSpecies>>&... tilde_p,
-      const Scalar<DataVector>& lapse,
-      const tnsr::i<DataVector, 3, Frame::Inertial>& d_lapse,
-      const tnsr::iJ<DataVector, 3, Frame::Inertial>& d_shift,
-      const tnsr::ijj<DataVector, 3, Frame::Inertial>& d_spatial_metric,
-      const tnsr::II<DataVector, 3, Frame::Inertial>& inv_spatial_metric,
-      const tnsr::ii<DataVector, 3, Frame::Inertial>&
-          extrinsic_curvature) noexcept {
+      const db::item_type<
```

Comment:
don't use `item_type` and `const_item_type` if you know the type

Reply:
@kidder : How do I do this while still expanding over all "NeutrinoSpecies"? 
I can't write 
const Scalar<DataVector>& source_n
as I want one source_n for each neutrino species.

Reply:
ah I see, @nilsdeppe any ideas?  at the very least they should be `const_item_type`